_posts/2018-02-28-swiftfall.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title:  Swiftfall
+date:   2018-02-28 21:25:00
+categories: projects
+description: A wrapper for an API called Scryfall.
+author: Braden Bowdish
+author-image: https://avatars2.githubusercontent.com/u/15185628?s=460&v=4
+author-bio: Lover of Swift and Magic. 3rd year member of Computer Science House.
+author-email: bmbowdish@csh.rit.edu
+author-social:
+  github: https://github.com/bmbowdish
+---
+
+# What is Swiftfall? 
+
+Swiftfall is an API wrapper written in Swift for the API Scryfall.
+
+[Scryfall](https://scryfall.com/docs/api) is an API which handles information about the card game Magic: The Gathering. 
+
+I like Magic the Gathering, but getting tons of information about Magic cards is really annoying, especially if you don't want to or don't know how to implement a JSON parser or make requests to a website. Many newer developers would like to combine their passions but it's not always easy. 
+
+Thankfully, [Scrython](https://github.com/nandascott/Scrython) does this already, but in Python. 
+
+### Why Swift Was Appealing
+1) Swift's use of optionals was really appealing. Sometimes you can't be absolutely sure about whether or not a request has been successful. Optionals allow a developer to handle those situations extremely safely. 
+2) Swift has a JSON decoder built in and it is appealing to have no dependencies.
+3) iOS, Apple TV, Apple Watch, and MacOS development is far easier in Swift
```

Comment:
This should still be `are`.

Reply:
Agreed w/ James :point_up:, looks good to merge once that's done, IMO.